app/build.gradle.kts

```diff
@@ -47,4 +48,10 @@ dependencies {
   androidTestImplementation(libs.androidx.junit)
   androidTestImplementation(libs.androidx.espresso)
   implementation(projects.extension)
-}
\ No newline at end of file
+}
+
+ktlint {
```

Comment:
Is it better to configure it this way? So all the sub modules could apply the ktlint

```
allprojects { project->
    project.plugins.apply("org.jlleitschuh.gradle.ktlint")
    project.tasks.withType(org.jlleitschuh.gradle.ktlint.tasks.BaseKtLintCheckTask).configureEach {
        it.workerMaxHeapSize.set("512m")
    }
    project.ktlint {
        version = "0.45.2"
        debug = false
        verbose = true
        android = true
        outputToConsole = true
        outputColorName = "RED"
        ignoreFailures = false
        enableExperimentalRules = true
        disabledRules = ["final-newline","import-ordering","spacing-around-angle-brackets"]
        reporters { reporter "html" }
        baseline = file("config/ktlint/baseline.xml")
        filter {
            exclude("**/generated/**")
            include("**/kotlin/**")
        }
    }
}
```

Reply:
Done